test/test_util.py

```diff
@@ -482,10 +493,12 @@ def test_parse_url_bytes_type_error(self):
             ({"disable_cache": True}, {"cache-control": "no-cache"}),
         ],
     )
-    def test_make_headers(self, kwargs, expected):
-        assert make_headers(**kwargs) == expected
+    def test_make_headers(
+        self, kwargs: Dict[str, Union[bool, str]], expected: Dict[str, str]
+    ) -> None:
+        assert make_headers(**kwargs) == expected  # type: ignore[arg-type]
```

Comment:
`‍‍‍‍test/test_util.py:499: error: Argument 1 to "make_headers" has incompatible type "**Dict[str, Union[bool, str]]"; expected "Optional[bool]"  [arg-type]`
`test/test_util.py:499: error: Argument 1 to "make_headers" has incompatible type "**Dict[str, Union[bool, str]]"; expected "Optional[str]"  [arg-type]`